pkg/storage/sqlstorage/accounts.go

```diff
@@ -38,6 +39,34 @@ func (s *Store) accountsQuery(p map[string]interface{}) *sqlbuilder.SelectBuilde
 		}
 	}
 
+	if balance, ok := p["balance"]; ok && balance.(string) != "" {
+
+		sb.Join(s.schema.Table("volumes"), "accounts.address = volumes.account")
+		balanceOperation := "volumes.input - volumes.output "
+
+		balanceValue, err := strconv.ParseInt(balance.(string), 10, 0)
+		if err != nil {
+			panic(err)
+		}
+
+		if balanceOperator, ok := p["balance_operator"]; ok && balanceOperator.(string) != "" {
+			switch balanceOperator.(string) {
+			case "lte":
+				sb.Where(sb.LessEqualThan(balanceOperation, balanceValue))
+			case "lt":
+				sb.Where(sb.LessThan(balanceOperation, balanceValue))
+			case "gt":
+				sb.Where(sb.GreaterThan(balanceOperation, balanceValue))
+			case "e":
+				sb.Where(sb.Equal(balanceOperation, balanceValue))
+			default: // default to gte
```

Comment:
If the operator is invalid, I think that we should return a bad request error to the user instead of defaulting.

Reply:
yes, you're right for the operator, i'll fix it